app/services/sla_breach_service.rb

```diff
@@ -11,11 +11,27 @@ def breach_threshold_date
   # attention_needed_since is present and is before breach threshold
   # returns data in the format { vita_partner_id: breach_count }
   def attention_needed_breaches
-    Client.sla_tracked.group(:vita_partner_id).where(Client.arel_table[:attention_needed_since].lteq(breach_threshold_date)).count(:id)
+    Client.sla_tracked
+      .where(Client.arel_table[:attention_needed_since].lteq(breach_threshold_date))
+      .count(:id)
+      .group(:vita_partner_id)
   end
 
   # clients who messaged us and have not been responded to _with a message, call or email_ within the breach threshold
   def outgoing_communication_breaches
-    Client.sla_tracked.group(:vita_partner_id).where(Client.arel_table[:first_unanswered_incoming_interaction_at].lteq(breach_threshold_date)).count(:id)
+    Client.sla_tracked
+        .where(Client.arel_table[:first_unanswered_incoming_interaction_at].lteq(breach_threshold_date))
+        .group(:vita_partner_id).
+        .count(:id)
+  end
+
+  # clients who've messaged us _and we've not interacted with their profile at all_ within the breach threshold.
```

Comment:
The previous metric measures whether we'd replied to them. but there are other interactions we can make with their profile.